packages/examples/src/api/fhir/operations/rotate-client-secret.ts

```diff
@@ -0,0 +1,23 @@
+import { MedplumClient } from '@medplum/core';
+import { ClientApplication } from '@medplum/fhirtypes';
+import { randomUUID } from 'node:crypto';
+
+const medplum = new MedplumClient();
+
+const clientApplication: ClientApplication & { id: string } = {
+  resourceType: 'ClientApplication',
+  id: randomUUID(),
+  secret: randomUUID(),
+};
+
+// start-block rotate
+const result = await medplum.post(
+  medplum.fhirUrl('ClientApplication', clientApplication.id, '$rotate-secret').toString(),
```

Comment:
Small nit, don't need the `.toString` since `medplum.post` accepts a `string | URL`